flatpak/manifest_tool/tests/flutter/test_patches_edge_cases.py

```
@@ -4,7 +4,7 @@
 
 from pathlib import Path
 
-import pytest
+import pytest  # noqa: F401
```

Ruff (0.13.1)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)